docs/Installation.md

```diff
@@ -123,7 +123,7 @@ To install the `mlagents` Python package, activate your virtual environment and
 run from the command line:
 
 ```sh
-pip3 install mlagents
+pip3 install mlagents==0.16.1
```

Comment:
Not sure if this document is supported but `ml-agents/docs/Installation-Anaconda-Windows.md` also mentions installing mlagents and maybe needs a fixed version as well...

Reply:
Good catch, updated that too.